src/wizard/index.tsx

```diff
@@ -25,16 +26,22 @@ export interface WizardProperties {
 	steps: Step[];
 }
 
+export type StepRenderer = (
+	status: StepStatus | undefined,
+	index: number,
+	step: Step
+) => RenderResult;
+
 export interface WizardChildren {
 	/** Custom renderer for the wizardSteps, receives the checkbox group middleware and options */
-	step?(status: StepStatus | undefined, index: number, step: Step): RenderResult;
+	step?: StepRenderer;
 }
 
 export type StepStatus = 'pending' | 'inProgress' | 'complete' | 'error';
 
 const factory = create({ theme })
 	.properties<WizardProperties>()
-	.children<WizardChildren | undefined>();
+	.children<WizardChildren | RenderResult | undefined>();
```

Comment:
I think this needs to be `(RenderResult | RenderResult[])` otherwise going to run into problems with tsx